app/lib/services/voip/mockCall.ts

```
@@ -85,4 +112,4 @@ export function seedMockCall(overrides: MockCallOverrides = {}): void {
 export function launchMockCallView(overrides: MockCallOverrides = {}): void {
 	seedMockCall(overrides);
 	Navigation.navigate('CallView');
```

ESLint

[error] 'Navigation' is not defined.

(no-undef)